typescript/web/next-sitemap.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  siteUrl: process.env.PUBLIC_SITE_URL,
+};
```

Comment:
Although its name is not ideal for on-premise, the `VERCEL_URL` variable is already used in our code.

```suggestion
const { VERCEL_URL } = process.env;

if (!VERCEL_URL) {
  throw new Error("Missing required environment variable VERCEL_URL");
}

const SITE_URL =
  VERCEL_URL.includes("://")
    ? VERCEL_URL
    : `https://${VERCEL_URL}`;

module.exports = { siteUrl: SITE_URL };
```

I don't like the optional addition of the `https` prefix at the beginning of `VERCEL_URL` but [that's the way it's expected to be handled](https://github.com/labelflow/labelflow/blob/4b37dbc44ebcd22d103db4ce6f887ee9e4df7cbc/typescript/web/src/constants.ts#L3) for now since it's defined with the prefix our `development.env` file for example.
As such let's assume that it's the way to go and it'll be cleaned once we finish the on-premise PR.

Reply:
Fixed.

By the way, I saw the file you linked the other day and it's written `labelflow.com`, do we own this domain ?

Reply:
Result of `whois labelflow.com` makes me think that we don't own it yet 😕
@nicOwlas is that normal?

Reply:
Yes we don't own it yet. Soon ;-)
https://supabase.com/blog/2021/04/02/supabase-dot-com